tests/foxbox-simulator/http-server.js

```diff
@@ -13,4 +15,10 @@ foxboxSimulator.get('/ping', (req, res) => {
   res.status(204).end();
 });
 
-foxboxSimulator.listen(3000);
+https.createServer(
+  {
+    key: fs.readFileSync(path.join(__dirname, '../../certs/key.pem')),
+    cert: fs.readFileSync(path.join(__dirname, '../../certs/cert.pem')),
```

Comment:
Why are the tests linted differently from the rest of the code? The `.eslintrc` file in `test/` should extend the one at the root level.